gitensite/apps/content/views.py

```diff
@@ -38,3 +43,15 @@ def get_queryset(self):
             return super(AjaxListView,self).get_queryset().filter(title__icontains=self.request.GET['q'])
         else:
             return super(AjaxListView,self).get_queryset()
+
+class BookPostView(TemplateView):
+    @method_decorator(csrf_exempt)
```

Comment:
We had to use the method decorator in order to disable Django's CSRF protection feature which was preventing us from implementing this POST endpoint. It is my understanding that the CSRF feature is meant to be used with HTML forms, so I don't think we need it.

Reply:
You're correct; what I meant was that you can use csrf_exempt as a decorator directly.

Reply:
Can you provide an example of what decorating a view in urls.py would look like?

Reply:
in https://github.com/EbookFoundation/regluit/blob/master/frontend/urls.py#L108
csrf_exempt is in urls.py because the view is defined by an inherited method of a view class

in https://github.com/EbookFoundation/regluit/blob/master/api/views.py#L105
csrf_exempt decorates the view directly where it is defined